src/util/format.ts

```diff
@@ -227,7 +233,7 @@ export function getTooltipMarker(inOpt: ColorString | GetTooltipMarkerOpt, extra
             // Only support string
             + encodeHTML(color) + ';' + (extraCssText || '') + '"></span>'
         : '<span style="display:inline-block;margin-right:4px;'
-            + 'border-radius:10px;width:10px;height:10px;background-color:'
+            + 'border-radius:' + size + ';width:' + size + ';height:' + size + ';background-color:'
```

Comment:
There are two `rendermode` for `tooltip`: `html` and `canvas`. You should also change code in `canvas` mode. just below it.

Reply:
im not sure if I should do the same for the subitem.